apistructs/pipeline_label.go

```diff
@@ -52,3 +56,32 @@ type TargetIDSelectByLabelRequest struct {
 	// OrderByTargetIDAsc 根据 target_id 升序，默认为 false，即降序
 	OrderByTargetIDAsc bool `json:"orderByTargetIDAsc"`
 }
+
+type PipelineLabelBatchInsertRequest struct {
+	Labels []PipelineLabel `json:"labels"`
+}
+
+type PipelineLabelListRequest struct {
+	PipelineSource  PipelineSource `schema:"pipelineSource" json:"pipelineSource"`
+	PipelineYmlName string         `schema:"pipelineYmlName" json:"pipelineYmlName"`
+	TargetIDs       []uint64       `schema:"targetIds" json:"targetIds"`
+	MatchKeys       []string       `schema:"matchKeys" json:"matchKeys"`
```

Comment:
Same issue.
Use `matchKey` in schema and this library will collect all values with same key in query params to `MatchKeys` slice.